cms/src/test/java/uk/nhs/digital/ps/modules/searchableFlagModule/JcrProvider.java

```diff
@@ -0,0 +1,73 @@
+package uk.nhs.digital.ps.modules.searchableFlagModule;
+
+import static java.util.Collections.emptyMap;
+
+import org.apache.sling.testing.mock.jcr.MockJcr;
+import org.yaml.snakeyaml.Yaml;
+
+import javax.jcr.Node;
+import javax.jcr.Repository;
+import javax.jcr.RepositoryException;
+import javax.jcr.Session;
+import java.io.InputStream;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Map;
+
+public class JcrProvider {
+
+    public Session getJcrFromFixture(InputStream fixture) {
+        Repository repository = null;
+        Session session = null;
+        try {
+            repository = MockJcr.newRepository();
+            session = repository.login();
+            Node rootNode = session.getRootNode();
+
+            Yaml parser = new Yaml();
+            List<Map<String, Object>> nodes = (ArrayList<Map<String, Object>>) parser.load(fixture);
+
+            nodes.forEach((object) -> {
+                NodeYaml node = new NodeYaml(
+                    (String) object.get("path"),
+                    (String) object.get("primaryType"),
+                    (Map<String, Object>) object.getOrDefault("properties", emptyMap())
+                );
+                try {
+                    Node n = rootNode.addNode(node.path, node.primaryType);
+                    node.properties.forEach((key, value) -> {
+                        try {
+                            if (value instanceof Boolean) {
+                                n.setProperty(key, (Boolean) value);
+                            } else {
+                                n.setProperty(key, value.toString());
+                            }
+                        } catch (RepositoryException e) {
+                            e.printStackTrace();
```

Comment:
Should throw these really as runtimeExceptions so the tests fail if this happens.